jnigen/tool/regenerate_bindings.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Run this script after any change which affects generated bindings.
```

Comment:
// Run this script after any change which affects generated bindings.
//
// This will update all generated bindings in the whole repository.